src/warehouse/v0/util.js

```diff
@@ -6,7 +6,7 @@ const toSnakeCase = (str) => {
     return '';
   }
   return String(str)
-    .replace(/^[^A-Za-z0-9]*|[^A-Za-z0-9]*$/g, '')
+    .replace(/^\W+|\W+$/g, '')
```

Comment:
W essentially includes `[a-zA-Z0-9_]` right. It does not match with the previous implementation.